tests/resources/dsl/botched_deployment.yaml

```diff
@@ -0,0 +1,11 @@
+tosca_definitions_version: cloudify_dsl_1_2
```

Comment:
on previous tasks, i was advised to create a blueprint for each scenario, so they'll be independent of each other. imo, i can use basic.yaml here it doesn't matter